lib/static.js

```diff
@@ -14,6 +14,8 @@ const files = new Set();
 
 const mdImageRE = /!\[[^\]]*\]\((.*?)\s*(["'](?:.*[^"'])["'])?\s*\)/g;
 
+const htmlImageRE = /<img.+src=["'](.+?)["'](?:.+?)>/g;
```

Comment:
Based on https://stackoverflow.com/a/57223323/4316405. RE is not a great tool to parse (X)HTML, I know, but here we know exactly what we are looking for.